modules/ndu/src/backend/conditions.ts

```diff
@@ -146,5 +146,47 @@ export const dialogConditions: sdk.Condition[] = [
 
       return foundMatch ? 1 : 0
     }
+  },
+  {
+    id: 'workflow_ended',
+    label: 'The user ended a workflow',
```

Comment:
Yup you are right, but since NDU is pretty much a work in progress and the vocabulary may change, I think it make sense to keep the english words for now. 

Just extracting the existing language for all existing conditions & parameters will be a huge PR in itself

Reply:
ok let's keep this in mid